QuickOSM/core/process.py

```diff
@@ -23,7 +24,7 @@
 from QuickOSM.core.query_factory import QueryFactory
 from QuickOSM.core.query_preparation import QueryPreparation
 from QuickOSM.core.utilities.tools import get_default_encoding, get_setting
-from QuickOSM.definitions.format import Format
+import QuickOSM.definitions.format as Format
```

Comment:
It doesn't work `from QuickOSM.definitions.format import Format` ?

Reply:
It was for some test, it's back to normal